benchmarks/benchmark/tools/locust-load-inference/locust-docker/locust-tasks/tasks.py

```diff
@@ -300,3 +306,78 @@ def total_content_length():
             Add a route to the Locust web app, where we can see the total content-length
             """
             return local_metric_collector.json_dump_report()
+
+class GrpcUser(User):
+    abstract = True
+    stub_class = None
+
+    def __init__(self, environment):
+        super().__init__(environment)
+        for attr_value, attr_name in ((self.host, "host"), (self.stub_class, "stub_class")):
+            if attr_value is None:
+                raise LocustError(f"You must specify the {attr_name}.")
+
+        self._channel = grpc.insecure_channel(self.host)
```

Comment:
Is the benchmarking done synchronously or asynchronously? (benchmarking done on the Jetstream repo uses grpc aio - https://github.com/google/JetStream/blob/main/benchmarks/benchmark_serving.py#L374

Reply:
They do. But the only reference of it is awaiting: https://github.com/google/JetStream/blob/main/benchmarks/benchmark_serving.py#L407

Also, I'm not sure we want to have asynchronous behavior for benchmarking atm. It could lead to unexpected results. I think our solution currently would be to spin up more workers.